server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -78,6 +77,8 @@
 
   private TsFileResource tsFileResource;
 
+  private long timeRange;
```

Comment:
It is weird that you call a single number 'range'. Please add comments to this field or rename it.

Reply:
fixed~